src/lib/testPaymentExpiry.js

```diff
@@ -87,7 +87,7 @@ TransferTester.prototype.validateMaxHoldTime = function () {
 
   if (expiresAt) {
     this.validateExpiryHoldTime(expiresAt)
-  } else {
+  } else if (this.destinationTransfer.executionCondition) {
```

Comment:
I don't think this change is needed (see https://github.com/interledger/five-bells-connector/pull/180).